data_collection/timeprof_matrix_bot.py

```diff
@@ -471,24 +471,26 @@ def create_next_sample_time(self, prev_sample_time, rate):
         next_sample_time = prev_sample_time + timedelta(minutes=interval)
         return next_sample_time
 
-    async def schedule_next_sample(self, user_id, sample_time):
+    async def ping(self, user_id, sample_time):
```

Comment:
If this is to be renamed, it has to be taken into account in multiple places. However, I don't like the name "ping".